src/lambda/evaluation_context.rs

```diff
@@ -1,19 +1,46 @@
 use std::time::Duration;
 
 use async_graphql::dynamic::ResolverContext;
-#[allow(unused_imports)]
-use async_graphql::InputType;
+use async_graphql::{Name, Value};
 use derive_setters::Setters;
+use indexmap::IndexMap;
 use reqwest::header::HeaderMap;
 
 use crate::http::RequestContext;
 
+pub trait GraphqlContext<'a> {
```

Comment:
```suggestion
pub trait ResolverContextLike<'a> {
```
Just being explicit about the name will make it easier to understand.

- [ ] We should move it to it's own file.

Reply:
done